imageroot/actions/restore-module/06copyenv

```diff
@@ -23,3 +23,4 @@ for evar in [
 ]:
     if evar in original_environment:
         agent.set_env(evar, original_environment[evar])
+agent.dump_env()
```

Comment:
this is obsolete now and does not more than a warning in logs, you can remove it safely, sometimes I forgot to remove it